web_source/cities/asia/china/chengdu.md

```diff
@@ -4,12 +4,15 @@ Country: China (Asia)
 
 ##### Also known as:
 
-FIXME
+成都, Chentu, Ch'en-tu, Chengtu, Ch'eng-tu
 
 ### What we would like to see there
 
-FIXME
+The Chengdu Research Base of Giant Panda Breeding (or simply Chengdu Panda Base). This is a public non-profit breeding and research institute for giant pandas, red pandas, and other rare animals.
+
```

Comment:
Great to see that Chengdu is the home of panda breeding. Is there anything else to do in the city?